### ISSUE TITLE
ATM name field tagging

### ISSUE THREAD

**Issue by peternewman** [osmlab/name-suggestion-index#5338]
I don't know if this is tagging for the renderer, but I see iD now offers to upgrade ATMs using the templated tagging.

If you do this with say the bank Foo, then the ATM also gets the tag `name=Foo` which seems a bit confusing, and doesn't match the behaviour of other systems as @bhousel screenshotted before:
https://github.com/osmlab/name-suggestion-index/issues/4975#issuecomment-803634583

It also means that in things like the commit changes bit of iD it's just listed as Foo, rather than Foo ATM which it feels to me like it should be?

Although I guess a different word may want to be used in other countries (perhaps even Cash Machine in the UK).

I see there was further discussion in:
https://github.com/osmlab/name-suggestion-index/issues/2883#issuecomment-726160004
https://github.com/osmlab/name-suggestion-index/commit/46ab8eba06f31571b02039883f59ec5a6204ec00#commitcomment-44125210

So maybe I should be raising this elsewhere, or it's a non issue. It sort of makes more sense on the rendered map then when displayed in ID.

**Comment by bhousel** [comment]
I don't have really strong opinions about what the ATMs are named.. 

**What we do currently** is just use a template item to copy over all the tags from the corresponding banks, and add `operator:*` tags too.

https://github.com/osmlab/name-suggestion-index/blob/6e4902ceb8b46d4588448a1df4c335a7e3c3b2f8/data/brands/amenity/atm.json#L8-L16


For example: https://nsi.guide/index.html?t=brands&k=amenity&v=atm#barclays-2f6feb
```js
amenity=atm
brand=Barclays
brand:wikidata=Q245343
brand:wikipedia=en:Barclays
name=Barclays
operator=Barclays
operator:wikidata=Q245343
operator:wikipedia=en:Barclays
```

There is definitely a bunch of tag redundancy here, but I don't really see this as a big problem.  These tags give renderers a lot of options on how to display ATMs - anything from "no name" to "literal name of bank" to "name of bank + 'ATM'" 

I'm inclined to leave it alone unless it's causing a real problem for someone.

**Comment by peternewman** [comment]
Yeah maybe it's as I was thinking as I wrote it and it should be raised in the renders/iD etc to append the type of object to the name, although I suspect that's not consistent in all cases e.g. https://www.mylondon.news/news/south-london-news/name-new-battersea-power-station-21615110
